en/orm/table-objects.rst

```diff
@@ -202,6 +202,17 @@ afterSave
 
 The ``Model.afterSave`` event is fired after an entity is saved.
 
+afterSaveCommit
+---------------
+
+.. php:method:: afterSaveCommit(Event $event, Entity $entity, ArrayObject $options)
+
+The ``Model.afterSaveCommit`` event is fired after the transaction in which save
```

Comment:
better "... in which __a__ save operation is wrapped..."?
or 
"... in which __the__ save operation is wrapped..."?